lib/t/matches.ex

```diff
@@ -593,4 +599,26 @@ defmodule T.Matches do
       broadcast_for_user(uid, message)
     end
   end
+
+  def match_check() do
+    MatchEvents
+    |> distinct([m], m.match_id)
+    |> join(:inner, [m], ma in Match, on: ma.id == m.match_id)
+    |> where([m], m.timestamp < fragment("now() - INTERVAL '48 hours'"))
+    |> select([m, ma], {m.match_id, ma.user_id_1})
+    |> T.Repo.all()
+    |> Enum.map(fn {match_id, user_id} ->
+      T.Matches.unmatch_match(user_id, match_id)
```

Comment:
`unmatch_match` func was made to unmatch one user from another (initiated by other user). here we need a similar function but for expire_match:
* it will do similar stuff in db (delete_match and delete_likes)
* but it will broadcast different event (not `notify_unmatch`, but rather `notify_expired`
* also (minor) - logger message will be different here as well

Reply:
@orlovvlad95 on a separate note - I just realised that we will need a separate pruner for "soon to become expired matches" for us to send push-notifications in advance (sth like "your match is about to expire, jump in an schedule an audio date")